### PR TITLE
Fix some minor misspells using misspell

### PR DESCRIPTION
**Pull request by aerostitch**
**What this PR does / why we need it**:
Fixes some misspells detected with misspell that might become handy when merging #59463 .
I saw that other files were already taken care of in other PR.
Added those  I didn't see fixed here.

**Special notes for your reviewer**:

I'm not 100% sure if the files in the api dir are supposed to be updated manually. I can revert that part if not.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by aerostitch** [comment]
As per bot's request:
/assign @wojtek-t

**Comment by oomichi** [comment]
/lgtm
trivial fix :-)

**Comment by steveperry-53** [comment]
@aerostitch Some of the files in this PR are generated, so you shouldn't be changing them directly.

File: CHANGELOG-1.10.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ filename | sha256 hash
...
 * `kubectl scale` can now scale any resource (kube, CRD, aggregate) conforming to the standard scale endpoint ([#58298](https://github.com/kubernetes/kubernetes/pull/58298), [@p0lyn0mial](https://github.com/p0lyn0mial))
-* kube-apiserver flag --tls-ca-file has had no effect for some time.  It is now deprecated and slated for removal in 1.11.  If you are specifying this flag, you must remove it from your launch config before ugprading to 1.11. ([#58968](https://github.com/kubernetes/kubernetes/pull/58968), [@deads2k](https://github.com/deads2k))
+* kube-apiserver flag --tls-ca-file has had no effect for some time.  It is now deprecated and slated for removal in 1.11.  If you are specifying this flag, you must remove it from your launch config before uprading to 1.11. ([#58968](https://github.com/kubernetes/kubernetes/pull/58968), [@deads2k](https://github.com/deads2k))
 * Fix regression in the CRI: do not add a default hostname on short image names ([#58955](https://github.com/kubernetes/kubernetes/pull/58955), [@runcom](https://github.com/runcom))
```
Comment (steveperry-53) [inline CHANGELOG-1.10.md]: uprading -> upgrading

Comment (aerostitch) [reply CHANGELOG-1.10.md]: Sorry for that. Fixing it will push when removing the generated files from the commits.

**Comment by aerostitch** [comment]
Thanks for the review @steveperry-53 . You're talking about the files in the `api` directory right?

Thanks
Joseph

**Comment by aerostitch** [comment]
Removed the changes to the swagger and openapi files. There should be no more generated files if I caught them correctly.

**Comment by steveperry-53** [comment]
@aerostitch Yes, I was talking about files in the api directory.

**Comment by aerostitch** [comment]
Thanks @steveperry-53 . I removed them.

**Pull request merged**